### ISSUE TITLE
autocomplete rule should ignore readonly fields

### ISSUE THREAD

**Issue by WilcoFiers** [dequelabs/axe-core#4708]
Axe-core's autocomplete-valid rule should not fail on readonly fields. If a field is readonly it cannot be used for user input.

```html
<input readonly autocomplete="some invalid value" />
```
